Backpack/CardButton/Classes/SaveCardButton.swift

```diff
@@ -0,0 +1,273 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+public class BPKSaveCardButton: UIButton {
+
+    public var checked: Bool {
```

Comment:
Should we use the native 'isSelected'
This would make it more intuitive to set a selected/unselected state.

Reply:
isSelected is used now in 514d5cb8f8720de2e4fa73bfa61be3fdc26b03f5